topics/activitypub/index.md

```diff
@@ -0,0 +1,11 @@
+---
+display_name: ActivityPub
+github_url: https://github.com/w3c/activitypub
+logo: activitypub.png
+related: decentralized, federated, protocol, social-network, w3c
```

Comment:
Could add `api` for users who see API mentioned in the description and want to know more about that.